src/appMain/smartDeviceLink.ini

```diff
@@ -37,10 +37,10 @@ LogsEnabled = true
 AppConfigFolder =
 ; Contains SDL configuration files - .json/.ini
 ; Default value is SDL working directory
-AppStorageFolder = storage
+AppStorageFolder = /tmp/sdl/storage
 ; Contains resourses, e.g. audio8bit.wav
 ; Default value is SDL working directory
-AppResourceFolder =
+AppResourceFolder = /usr/local/bin
```

Comment:
@alexkutsan I see many paths changes in ini. Not sure if all of them are applicable/best for Linux environment.
Maybe it makes sense to create a separate ini file for a deb package and replace it when DEB-package build is requested?

Reply:
There are several tricks in cmake to perform such staff. I will try to implement